bundle/compliance/cis_gcp/test_data.rego

```diff
@@ -45,3 +57,10 @@ not_eval_resource = {
 	"type": "key-management",
 	"subType": "no-exisitng-type",
 }
+
+# missing resource.iam_policy
+no_policy_resource = {
```

Comment:
this is duplicated, will change to `generate_no_policy_resource` in next rule that uses it